static/js/tools/timeline/util.ts

```diff
@@ -56,7 +56,8 @@ export function setTokensToUrl(tokens: TokenInfo[]): void {
   window.location.hash = urlParams.toString();
 }
 
-// add one statVar
+// Add a token to the url. A token could either be a place dcid, a stat var or
+// a [stat var,denominator] pair separated by comma.
```

Comment:
from the landing page api, it looks like [stat var, denominator] pair is now separated by "|" instead of comma?

Reply:
Yes, "," is used as the separator for places, want to have a different symbol to differentiate here. @pradh can you confirm is there "|" used in stat var dcid?

Reply:
It turns out bio DCIDs do contain '|', but I do want to get rid of it... this may be fine.